documentation/source/swagger/swagger_consents.yaml

```
@@ -70,7 +70,13 @@ paths:
                 $ref: '#/components/schemas/ResponseError'
       security:
         - OAuth2Security:
-            - "consents"
+            - accounts 
+            - customers 
+            - credit-cards-accounts 
+            - loans 
+            - financings 
+            - unarranged-accounts-overdraft 
+            - invoice-financings
```

Essa API só precisa do escopo de consents
```suggestion
            - consents
```

---

OK! Sem problemas